sphinx/api/command-reference.rst

```diff
@@ -9,6 +9,59 @@ Rose Commands
 
 .. auto-cli-doc:: rose rose
 
+----
+
+.. _command-rose-test-battery:
```

Comment:
The command reference is auto-generated & should now be excluded from the content under version control. If you have added new description for a command, it needs to be put with that command's module file so it gets printed to STDOUT with the ``--help`` option, & then the script to parse will pick it up & add it to this rendered file in the HTML docs.

Reply:
@sadielbartholomew this is here because the command has been moved to `etc/` so the `.. auto-cli-doc:: rose rose` directive is no longer able to document it so Tim has added in the derived content for just the test battery command for now (as there are references to it elsewhere in the docs).

Reply:
Thanks for the explanation. I'll add a tick-box note to the main PR comment as a reminder to create a follow-on Issue for this. (@wxtim in future could important background like this be popped onto the main PR comment to aide with reviewing?)

Reply:
Yes. sorry.